features/admin_creates_auction.feature

```diff
@@ -18,6 +18,7 @@ Feature: Admin creates auctions in the admins panel
     Then I should see the start time I set for the auction
     And I should see the end time I set for the auction
 
+  @freeze_time
```

Comment:
Me too. I am looking into it so I can understand exactly.

Also, this needs a Dr. Horrible reference